addons/binding/org.openhab.binding.heos/ESH-INF/thing/HeosPlayer.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Heos Player Thing Type -->
+	<thing-type id="player">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>Heos Player</label>
+		<description>A HEOS Player of the HEOS Network</description>
+
+		<channels>
+			<channel id="Control" typeId="control" />
+			<channel id="Volume" typeId="volume" />
+			<channel id="Mute" typeId="mute" />
+			<channel id="Inputs" typeId="input" />
+			<channel id="Title" typeId="title" />
+			<channel id="Artist" typeId="artist" />
+			<channel id="Album" typeId="album" />
+			<channel id="Cover" typeId="cover" />
+			<channel id="CurrentPosition" typeId="currentPosition" />
+			<channel id="Duration" typeId="duration" />
+			<channel id="Type" typeId="type" />
+			<channel id="Station" typeId="station" />
+			<channel id="PlayUrl" typeId="playUrl" />
+			<channel id="Shuffel" typeId="shuffleMode" />
+			<channel id="RepeatMode" typeId="repeatMode" />
+			<channel id="Playlists" typeId="playlists" />
+		</channels>
+		<config-description>
```

Comment:
Same here. At least name & model must not be parameters, but only properties of the Thing.
What is the normal way to identify the device? It's Player ID or its IP address? Ideally, only one of those should be defined in the config.